### ISSUE TITLE
Implement PRISMA Schema for Consent Questions Table

### ISSUE THREAD

**Issue by b-f-chan** [OHCRN/platform#53]
**EXIT CRITERIA:**

- [ ] PRISMA schema exists for the consent questions table within the Consent DAS
- [ ] Provide seed data for this table
- [ ] Basic CRUD operations exist in the Consent DAS

**Comment by b-f-chan** [comment]
On hold --> Blocked by DAS ticket for now; moving to J-Browse for now

**Comment by mistryrn** [comment]
Leaving a note here for reference for this ticket, and for a future "Participant Response" schema ticket

Since adding the DASes deprecated the existing schema in `consent-api`, the old `schema.prisma` in `consent-api` has been deleted. For reference, here is what was there before:

Note that the model has changed since this was implemented, so it is **only intended as a Prisma/schema reference/example, but not the actual model to be implemented.**

```
model Participant {
  id       String @id @default(cuid())
  name     String
  email    String @unique
  participantResponses ParticipantResponse[]
}

enum ConsentCategory {
  INFORMED_CONSENT
  CONSENT_RELEASE_DATA
  CONSENT_RESEARCH_PARTICIPATION
  CONSENT_RECONTACT
  CONSENT_REVIEW_SIGN
}

model ConsentQuestion {
  id        String @id
  isActive  Boolean @default(true)
  createdAt DateTime @default(now())
  category  ConsentCategory
  participantResponses ParticipantResponse[]
}

model ParticipantResponse {
  id        String @default(cuid())
  participant Participant @relation(fields: [participantId], references: [id], onDelete: Restrict, onUpdate: Restrict)
  participantId    String
  consentQuestion ConsentQuestion @relation(fields: [consentQuestionId], references: [id], onDelete: Restrict, onUpdate: Restrict)
  consentQuestionId String
  @@id([id, participantId, consentQuestionId])
  response  Boolean
  submittedAt DateTime @default(now())
}
```

**Comment by b-f-chan** [comment]
@samrichca to do some handoff with Alyssa on this ticket for next sprint

**Comment by b-f-chan** [comment]
@alyssadsouza - @samrichca had a PR previously for this, please check if there is anything reusable there before starting
